---
layout: page
title: "FHIR Chat  · Validator Bug · implementers"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179166-implementers/index.html">implementers</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179166-implementers/topic/Validator.20Bug.html">Validator Bug</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="166759367"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179166-implementers/topic/Validator%20Bug/near/166759367" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Grahame Grieve <a href="https://fhir.github.io/chat-archive/stream/179166-implementers/topic/Validator.20Bug.html#166759367">(May 28 2019 at 21:05)</a>:</h4>
<p>I just found and fixed a serious issue in the validator: under some circumstances, the validator would fail to check value set membership for some coded values. Specifically, when validating against an IG that restricts the value set of a FHIR defined code system, the code would be accepted if was a valid code, not if it was valid according to the IG.</p>
<ul>
<li>Resources that may have been deemed to be valid in the past may now fail.</li>
</ul>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
